schemas/discovery-schema.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "https://zowe.org/schema/api-layer-config",
+    "title": "Discovery service configuration",
+    "description": "Properties that can be used to configure discovery service in a configuration file, such as zowe.yaml.",
+    "type": "object",
+    "additionalProperties": true,
+    "properties": {
+        "apiml": {
+            "type": "object",
+            "description": "",
+            "additionalProperties": false,
```

Comment:
this will mean no further `apiml.*` properties will be accepted - I don't think we want that in the schema

also, only `serviceIdPrefixReplacer` is added, there are other properties, what constitutes being worth added to this schema?

Reply:
I was using values from start.sh and there are no others, do you have some on mind that can be added?